From 78fac26891fe7494355021dbac109b807b8c6d53 Mon Sep 17 00:00:00 2001
From: Joachim Nilsson <troglobit@gmail.com>
Date: Mon, 14 May 2018 15:40:43 +0200
Subject: [PATCH] Convert AM_INIT_AUTOMAKE() --> AC_INIT() and trigger non-GNU
 project

Converts from old-style configure syntax with AM_INIT_AUTOMAKE to
AC_INIT and enable "foreign" mode, i.e. non-GNU conformant tree
to avoid copying in COPYING and other files when autogen.sh runs.

Signed-off-by: Joachim Nilsson <troglobit@gmail.com>
---
 configure.in | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/configure.in b/configure.in
index 2f72645..c5fbc4b 100755
--- a/configure.in
+++ b/configure.in
@@ -4,8 +4,8 @@ dnl     This file is an input file used by the GNU "autoconf" program to
 dnl     generate the file "configure", which is run during the build
 dnl     to configure the system for the local environment.
 
-AC_INIT
-AM_INIT_AUTOMAKE(fcgi, 2.4.1-SNAP-0910052249)
+AC_INIT(fcgi, 2.4.1-SNAP-0910052249)
+AM_INIT_AUTOMAKE([1.11 foreign])
 
 AM_CONFIG_HEADER(fcgi_config.h)
 
-- 
2.20.0

